packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: Console
+description: An overview of the Console experience's design guidelines.
+slug: /experiences/console/
+---
+
+import {graphql} from 'gatsby';
+
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Stack} from '@twilio-paste/stack';
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {ImageCaption} from '../../../components/ImageCaption';
+
+export const pageQuery = graphql`
+  {
+    mdx(frontmatter: {slug: {eq: "/experiences/console/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.EXPERIENCES}
+/>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Base principles
+
+- Do not duplicate Console content in different places
+- Focus on the user journey, not just your product or feature
+- Rely on existing patterns before creating a new one
+- Use Paste with the Default theme for all new content
+- Console navigation is optimized for findability, not promoting products or features
+
+## Develop vs. Monitor
+
+In our research, we found that a majority of users expect content to be organized by
+the phases of their development lifecycle, not by product vertical.
+A user's journey in building an application on Twilio starts with **Developing**,
+then moves into **Monitoring** after they launch their application.
+
+<Box marginBottom="space40">
+<Table tableLayout="fixed">
+<TBody>
```

Comment:
question (non-blocking): what happened to the indentation here? was prettier fighting it?

Reply:
This was intentional, but it wasn't Prettier- it wasn't compiling correctly (kept thinking that certain closing tags weren't present). I think it's because indentation has meaning in mdx, specifically that indenting text more than 3 times formats it to code.

Reply:
Weird, the Pagination page works and does something similar

Reply:
I think this was happening because I wasn't wrapping the content in Text or Paragraph. Now that I am, things work as expected. Thanks for bringing this up, @TheSisb!